includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -1096,7 +1096,7 @@ public function wc_setup_payment() {
 						)
 					),
 					esc_url( admin_url( 'admin.php?page=wc-settings&tab=checkout&section=paypal' ) ),
-					esc_url( admin_url( 'admin.php?page=wc-addons&section=payment-gateways' ) )
+					esc_url( admin_url( 'admin.php?page=wc-addons' ) )
```

Comment:
What does `&section=payment-gateways` do? If the rest of this was removed in a prior PR, can you provide a link?

Reply:
This used to link to a section in the addons admin page to the Woo Payment gateways which has since been flattend with CC notes.

Reply:
Link to prior PR please?

Reply:
PR #47 trimmed the content of the Addons page.